morph/react/block-list.js

```diff
@@ -49,21 +50,45 @@ export function enter(node, parent, state) {
   }
 
   state.render.push(
-    `{Array.isArray(${value}) && ${value}.map((item, index, list) => `
+    `{Array.isArray(${value}) && ${value}.map((item, index, list) => {`
   )
 
   if (state.viewPath) {
     let itemDataContextName =
       getProp(node, 'itemDataContextName') ||
       defaultItemDataContextName(node, from)
+    let keys
+    if (hasProp(node, 'itemKeys')) {
+      keys = getProp(node, 'itemKeys')
+        .value.split(',')
+        .map((key) => key.trim())
+    } else {
+      keys = ['id']
+
+      state.render.push(`
```

Comment:
To your comment around not shipping this to prod, bundlers will remove it because of NODE_ENV. However, what about writing that code that if `tools` is true in the morpher?

Reply:
Sorry, I don't think I follow your suggestion. Do you have any code sample or we can take that discussion offline?

Reply:
Sure we can have a chat. It'll be a check like this one https://github.com/viewstools/morph/blob/master/ensure-views-files.js#L28

Reply:
Makes sense, I added the check as suggested